finch.yaml

```diff
@@ -151,9 +151,14 @@ provision:
     sudo chown $USER /mnt/lima-finch
     mkdir -p /mnt/lima-finch/containerd
     mkdir -p /mnt/lima-finch/nerdctl
+    mkdir -p /mnt/lima-finch/cni-config
+    mkdir -p /mnt/lima-finch/cni-local
     mkdir -p ~/.local/share/nerdctl
+    mkdir -p ~/.config/cni
```

Comment:
Why don't we need to `mkdir -p` for `~/.local/share/containerd` and `~/.local/share/cni`?

Reply:
these both exist by default by the time this script is run, whereas the `cni` and `nerdctl` configs are only created when they're used. I suppose there's no harm in adding them to make sure they are created in case this changes later.

Reply:
> these both exist by default by the time this script is run, whereas the cni and nerdctl configs are only created when they're used.

I see.

> I suppose there's no harm in adding them to make sure they are created in case this changes later.

Agree. Maybe we can add them then. It can also lower cognitive overhead (i.e., we won't need to know which ones are only created when they're used; instead, we just make sure everything is there).